tests/ga/extension_emulator.py

```diff
@@ -216,9 +219,16 @@ def wrapped_func(cmd, *args, **kwargs):
 
                 if seq < int(match.group("seq")):
                     seq = int(match.group("seq"))
+            
+            status_file = os.path.join(os.path.dirname(cmd), "status", "{seq}.status".format(seq=seq))
+            
+            if return_value == 0:
+                status_contents = ExtensionEmulator._EXT_STATUS_SUCCESS
+            else:
+                status_contents = ExtensionEmulator._EXT_STATUS_ERROR_BASE.copy()
+                status_contents[0]["status"]["substatus"] = return_value
```

Comment:
Thanks for adding this! A small request, can you make the `substatus` a json instead with something like - 
`{'exit_code': return_value}` just to make it more clearer and easier to understand and to eventually test.